src/app/components/DataLoader/DataLoader.jsx

```diff
@@ -1,10 +1,73 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import Actions from '@Actions';
+import { ajaxCall } from '@utils';
+import appConfig from '@appConfig';
 
 class DataLoader extends React.Component {
+  constructor(props) {
```

Comment:
Might be good to do a short code walkthrough for this new Component about its intent and implementation.

Reply:
Sounds good!